frontend/src/components/templates/Login/index.js

```diff
@@ -74,16 +75,8 @@ export const loginUrl = `${BACK_HOST}/users/auth/api_gouv${hashToQueryParams({
 
 const LoginButtons = ({ isOnNewEnrollmentPage }) => (
   <div className="login-buttons">
```

Comment:
If you delete this component, mind the suppression of the css rule binded to `.login-buttons` if any.